backend/LeetWars.Builder/Services/ConsumeMessages.cs

```diff
@@ -7,20 +12,39 @@ namespace LeetWars.Builder.Services
     public class ConsumeMessages : BackgroundService
     {
         private readonly IConsumerService _consumerService;
-
-        public ConsumeMessages(IConsumerService consumerService)
+        private readonly ICodeRunManagerService _codeRunManagerService;
+        private readonly IProducerService _producerService;
+        public ConsumeMessages(IConsumerService consumerService, ICodeRunManagerService codeRunManagerService, IProducerService producerService)
         {
             _consumerService = consumerService;
+            _codeRunManagerService = codeRunManagerService;
+            _producerService = producerService;
         }
 
         protected override Task ExecuteAsync(CancellationToken stoppingToken)
         {
-            var handler = new EventHandler<BasicDeliverEventArgs>((model, args) =>
+            var handler = new EventHandler<BasicDeliverEventArgs>(async (model, args) =>
             {
                 var body = args.Body.ToArray();
-                var message = Encoding.UTF8.GetString(body).Replace('"', '\0');
+                var message = Encoding.UTF8.GetString(body);
+
+                var request = JsonConvert.DeserializeObject<CodeRunRequest>(message);
+
+                var languages = new List<string> { "C#", "Javascript" };
```

Comment:
do not hardcode the values, store them in one place in order to change or expand the application easier